bench/wtperf/wtperf.h

```diff
@@ -261,7 +271,7 @@ struct __config_thread {		/* Per-thread structure */
 	TRACK truncate;			/* Truncate operations */
 	TRACK truncate_sleep;		/* Truncate sleep operations */
 	TRUNCATE_CONFIG trunc_cfg;	/* Truncate configuration */
-
+	THROTTLE_CONFIG throttle_cfg;	/* Throttle configuration */
```

Comment:
I'd leave in the blank line, but move both of them above the tracking section.  The structure has blank lines between unrelated fields.